src/VS.Web.CG.Mvc/Templates/Identity/ScaffoldingReadme.txt

```diff
@@ -1,23 +1,21 @@
-The code for Identity UI has been generated under Areas/Identity.
-Content files have been added under wwwroot/Identity
+Support for ASP.NET Core Identity was added to your project
+- The code for adding Identity to your project was generated under Areas/Identity.
+- Content files have been added under wwwroot/Identity.
 
-However you may need to make additional changes to the Startup's ConfigureServices method:
+Configuration of the Identity related services can be found in the Areas/Identity/IdentityHostingStartup.cs file.
 
-1. Enable Areas support for RazorPages:
+If your app was previously configured to use Identity, then you should remove the entire call to AddIdentity from your
```

Comment:
Instead of `entire call to AddIdentity`, should it just be `call to AddIdentity`?